nextpy/frontend/templates/jinja/web/pages/base_page.js.jinja2

```diff
@@ -13,6 +13,7 @@ Purpose:
 {# ==============================
      LIBRARY IMPORTS BLOCK
 ============================== #}
+import { DispatchContext } from 'utils/context'
```

Comment:
Instead of doing it here, we should move this to default imports which are being used by the jinja template from utils.py file (most probably)